aspnetcore/signalr/hubs.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Use Hubs in ASP.NET Core SignalR
+author: rachelappel
+description: Learn how to use hubs in ASP.NET Core SignalR.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 03/23/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/hubs
+---
+
+# Use Hubs in SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel) and [Kevin Griffin](http://twitter.com/1kevgriff)
+
+## What is a SignalR Hub
+
+The SignalR Hubs API enables you to call methods on connected clients from the server. In server code, you define methods that are called by client. In client code, you define methods that are called from the server. SignalR takes care of everything behind the scenes that makes real-time client-to-server and server-to-client communications possible.
+
+## Configure SignalR hubs
+
+Since SignalR is middleware, a call to `services.AddSignalR` is required in the `ConfigureServices` method of the `Startup` class to register the service.
```

Comment:
"Since SignalR is **a** middleware"

Reply:
Also technically speaking you don't call `AddSignalR` because it's a middleware. You call `UseSignalR` because it's a middleware, `AddSignalR` sets up the services and is a pre-req for the middleware. Maybe something more like
"The SignalR middleware requires some services to be able to run, to set those up you can call `services.AddSignalR`..."